### PR TITLE
IAE-69318 Add actions class

### PR DESCRIPTION
**Pull request by julieduong22**

### CODE REVIEW

File: src/stylesheets/components/_actions.scss [hunk 1]
```diff
@@ -0,0 +1,33 @@
+.sds-button--actions {
```
Comment (cwolf10) [inline src/stylesheets/components/_actions.scss:1]: There is a class called `sds-button--action` which defines the current (wrong) look in _menu.scss. Is there a reason that class needs to be preserved? If not, that class should be updated rather than creating two, very similar classes.

File: src/components/03-components/01-actions/actions.njk [hunk 2]
```diff
@@ -7,6 +7,11 @@
       <span class="sr-only">Screen reader description</span>
     </button>
 
+    <button class="sds-button--actions">
```
Comment (cwolf10) [inline src/components/03-components/01-actions/actions.njk:10]: This is resulting in the existing in a button with the old action icon in it, placed next to the new action icon. If the old one is wrong, why not replace it with this new one?

File: src/components/03-components/01-actions/actions.config.yml [hunk 3]
```diff
@@ -2,10 +2,13 @@ title: Actions Menu
 status: wip
 preview: "@sam"
 context:
-    style: primary
-    label: action
-    icon:
-        icon: ellipsis
-        classes: fa-fw
-        dataTransform: grow-5
+  style: primary
```
Comment (cwolf10) [inline src/components/03-components/01-actions/actions.config.yml:5]: Was the old config broken due to whitespace? If not, please revert the changes in this file. No need to clutter up this PR and the file history with purely whitespace changes.

File: src/stylesheets/components/_card.scss [hunk 4]
```diff
@@ -189,6 +189,7 @@ $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width
   }
 
   &--vertical {
+    text-align: center;
```
Comment (cwolf10) [inline src/stylesheets/components/_card.scss:192]: This does not seem to be related to the action icon. If it not, it may be better to pull this into a separate PR to better track and explain the need for this change.

File: src/components/03-components/01-actions/actions.njk (hunk 2 above)
Comment (julieduong22) [reply src/components/03-components/01-actions/actions.njk:10]: I think Dave wants to keep both looks (blue and white).

File: src/stylesheets/components/_actions.scss (hunk 1 above)
Comment (julieduong22) [reply src/stylesheets/components/_actions.scss:1]: The sds-button--action class is kept for the blue action icon. I may combine and put them together in the _actions.scss file.

File: src/stylesheets/components/_card.scss (hunk 4 above)
Comment (julieduong22) [reply src/stylesheets/components/_card.scss:192]: This is for the vertical header. I will move it to a new PR.

File: src/stylesheets/components/_actions.scss (hunk 1 above)
Comment (cwolf10) [reply src/stylesheets/components/_actions.scss:1]: Okay so it sounds like the ask is more for a variant of the existing action icon rather than a replacement. In that case, it'd probably be better to start with the existing class and then adjust whatever styles are different with this variant(bg color, border thickness, etc). The benefit of this is that the common styles will stay in sync as additional changes are made and we don't have the buttons unintentionally drifting apart by devs missing that they need to update styles in two places.

If I'm misunderstanding and the ask is for essentially two different buttons, then it makes sense to not have common styles shared between then.

Regardless, I think the class name should be changed to something that clearly differentiates between the existing and new actions button. If it isn't, there may be more hassle when the next developer who needs an action button comes along and has to determine whether they want `.sds-button--action` or `.sds-button--actions`. It'd be easier to understand `.sds-button--action` or `.sds-button--action-light` if the new button is just a variant.

**Comment by cwolf10** [comment]
Hey @julieduong22, following up on this PR also. The updated naming + how the style renders looks fine. Still have the additional formatting in actions.config.yml that needs to be undone. Also have the changes in _card.scss that need to be pulled out to a separate PR. Get those updated and pull master in to resolve the merge conflict and I'm good with approving this.

File: src/components/03-components/01-actions/actions.config.yml [hunk 5]
```diff
@@ -8,4 +8,7 @@ context:
         icon: ellipsis
         classes: fa-fw
         dataTransform: grow-5
+
```
Comment (cwolf10) [inline src/components/03-components/01-actions/actions.config.yml:11]: Clear this extra spacing

**Review by cwolf10 — APPROVED** [review]
